docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Commencing 
+FROM ubuntu:18.04
+
+#ENV DEBIAN_FRONTEND=noninteractive
+#ENV TZ=US/Chicago
+#RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+## Updates and software that I like
+# Install vnc, xvfb in order to create a 'fake' display and firefox
+RUN apt-get update
+RUN apt-get install -y htop tmux
+RUN apt-get install -y vim=2:8.0.1453-1ubuntu1
+RUN apt-get install -y neovim
+RUN apt-get update
+RUN apt-get install -y ssh 
+RUN apt-get install -y curl
+RUN apt-get install -y python 
+RUN apt-get install -y gpg 
+RUN apt-get install -y dirmngr 
+RUN apt-get update --fix-missing
+RUN apt-get install -y openssl 
+RUN apt-get install -y libssl-dev 
+RUN apt-get install -y libcurl4-openssl-dev
+RUN apt-get install -y pandoc
```

Comment:
this will take a while.  i think there's a binary we can install instead?

https://pandoc.org/installing.html

Reply:
You're right, not needed.